Makefile

```diff
@@ -169,16 +173,19 @@ stop-etcd:
 
 .PHONY: clean
 ## Removes all .coverprofile files, the vendor dir, and .go-pkg-cache
-clean: clean-deepcopy-gen clean-bin
+clean: clean-bin
 	find . -name '*.coverprofile' -type f -delete
 	rm -rf vendor .go-pkg-cache
 
-clean-deepcopy-gen:
-	rm -f .deepcopy_gen
-	find $(TOP_SRC_DIR) -name zz_generated* -exec rm {} \;
-
+.PHONY: clean-bin
 clean-bin:
-	rm -rf $(BINDIR) .deepcopy_gen_exes
+	rm -rf $(BINDIR)
+
+.PHONY: clean-deepcopy-gen
+clean-deepcopy-gen:
```

Comment:
If this is just a developer convenience (it's not from anywhere) then you should add a comment stating that.